resources/js/admin.js

```diff
@@ -95,4 +95,17 @@ $(function() {
             }
         });
     });
+    $('option').mousedown(function(e) {
+        e.preventDefault();
+        var originalScrollTop = $(this).parent().scrollTop();
+        console.log(originalScrollTop);
+        $(this).prop('selected', $(this).prop('selected') ? false : true);
+        var self = this;
+        $(this).parent().focus();
+        setTimeout(function() {
+            $(self).parent().scrollTop(originalScrollTop);
+        }, 0);

```

Comment:
Why do we have to use a timeout with a delay of 0 seconds?

Reply:
Good question; removing the timeout function works for FF, Safari, but not Chrome.